pylint/checkers/utils.py

```diff
@@ -231,6 +232,8 @@
 )
 
 T_Node = TypeVar("T_Node", bound=nodes.NodeNG)
+Checker = TypeVar("Checker", bound=BaseChecker)
```

Comment:
```suggestion
CheckerT = TypeVar("CheckerT", bound=BaseChecker)
```

I think it is good practice to add a `T` to `TypeVars`.